packages/wix-ui-icons-common/scripts/build-icons.js

```diff
@@ -0,0 +1,32 @@
+const spawnSync = require('child_process').spawnSync;
+const path = require('path');
+
+const buildIconsFromLibrary = (libraryPath, keepColors) => {
+  const fromPath = path.join(libraryPath, '/raw');
+  const toPath = path.join(libraryPath, '/dist');
+  const keepIconsColors = keepColors ? '--keep-colors' : '--monochrome';
+  let error = false;
+  try {
+    spawnSync('svg2react-icon', ['--typescript', keepIconsColors, fromPath, toPath] );
+  }
+  catch(err){
+    error = true;
+    console.log(err);
+  }
+  if(!error){
+    console.log("finished icons build for " + libraryPath);
+  }
```

Comment:
Just a small code style remark.
This can be written like this:
```suggestion
  try {
    spawnSync('svg2react-icon', ['--typescript', keepIconsColors, fromPath, toPath] );
    console.log("finished icons build for " + libraryPath);
  }
  catch(err){
    console.log(err);
  }
```
just to spare the addition of the `error` variable 😁